From d5e7953c35d895f2bdf29eca102bf40ef9685bfe Mon Sep 17 00:00:00 2001
From: Chris Roberts <cpr420@gmail.com>
Date: Sat, 15 Jul 2023 21:12:07 -0600
Subject: disable file_types support in the rdef

NNN is a terminal application and doesn't handle Haiku style file references

diff --git a/misc/haiku/nnn.rdef b/misc/haiku/nnn.rdef
index 7e029fd..ce0b992 100644
--- a/misc/haiku/nnn.rdef
+++ b/misc/haiku/nnn.rdef
@@ -21,9 +21,11 @@ resource app_version {
 
 resource app_flags B_MULTIPLE_LAUNCH | B_BACKGROUND_APP | B_ARGV_ONLY;
 
+/*
 resource file_types message {
     "types" = "application/x-vnd.Be-directory"
 };
+*/
 
 resource vector_icon {
 	$"6E6369660805FF032E343602011202B8E0C339BAC3B985CFB8B7E549F5FF4994"
-- 
2.39.2

